### ISSUE TITLE
Complete intial Graphnet backend implementation

### ISSUE THREAD

**Issue by benbc** [opensafely-core/databuilder#752]
We are collaborating with Graphnet to build an OpenSAFELY-Graphnet backend. The project is described in more detail [here](https://docs.google.com/document/d/1vbXLoHbN4lk4EnH3MZF8Q6mLpeVVTn5rVkyjlJtHK4Y/edit#heading=h.f62pjjqv6g89).

Prior to this issue (once #749 has been completed) there is a minimal Graphnet backend sufficient to run the smoke test. The next piece of work is to extend that to provide all the data required by the Booster Effectiveness study which is the end goal of this project.

## Acceptance criteria

- [ ] All tables and columns required by the [booster effectiveness study](https://github.com/opensafely-core/databuilder/tree/main/tests/acceptance/comparative_booster_study) are provided by the Graphnet backend.
- [ ] The backend maps the tables to [the schema](https://docs.google.com/spreadsheets/d/17NB_qm7ktAr9GMse57mPOCowFP4cQCc2/edit#gid=1862721383) provided by Graphnet.

## Exclusions

Actually running the dataset definition against the Graphnet backend is not included in this issue. That will follow from this issue being completed.

**Comment by benbc** [comment]
This is blocked by https://github.com/opensafely-core/databuilder/issues/749.

**Comment by benbc** [comment]
@inglesp This is now unblocked. It's not urgent since we are still blocked on Graphnet infrastucture setup. I think this would be a really good issue for someone who is learning Data Builder since it's relatively straightforward but gives them good exposure to real-world usage of the table definition stuff.

**Comment by benbc** [comment]
The Graphnet/CIPHA project is on hold until they've sorted their infrastructure out.
